wiki/Tournaments/Official_support/en.md

```diff
@@ -187,7 +187,7 @@ We have historically had issues where unscrupulous organisers have geared tourna
 
 ### Can a player who has been eliminated from play in my tournament join the staff after the fact and not cause problems? {#player-join-staff}
 
-No. Players who have been eliminated from the tournament/contest may not be enlisted as staff or assist in any organisational capacity with the sole exception of setting replays for mappool showcases. They can, however, be enlisted as a helper.
+Yes. Players who have been eliminated from a tournament may be enlisted as staff to assist in any non-managerial position.
```

Comment:
The previous text was very confusing so I agree the new version is better. But what is "non-managerial"? It's probably best to clarify since I can't understand this.

Reply:
This one was mostly turned this way as non-managerial position could differ depending on the structure of the tournament.
We probably could add most roles that are counted that most tournaments have, and just tell anyone with a different role to contact if required, as I'd assume they wouldn't be in this much cases.

Reply:
Why not link to the same anchor as the other one above?

Reply:
Discussed with PooN a bit, we decided to add the list of staff members that is above that question as it makes more sense anyway, since other roles aren't gonna be coming up this often.